server/src/main/java/org/opensearch/cluster/routing/allocation/allocator/BalancedShardsAllocator.java

```diff
@@ -256,6 +257,11 @@ float weight(Balancer balancer, ModelNode node, String index) {
         private final Metadata metadata;
         private final float avgShardsPerNode;
         private final NodeSorter sorter;
+        private final Set<RoutingNode> inEligibleTargetNode;
+        private int shardsProcessed;
+        private final List<Integer> targetNodesProcessed;
```

Comment:
Do we need this. Maybe it was used for logging earlier

Reply:
Removed except inEligibleTargetNode